agent/api/statechange.go

```diff
@@ -138,6 +140,7 @@ func NewContainerStateChangeEvent(task *apitask.Task, cont *apicontainer.Contain
 	event = ContainerStateChange{
 		TaskArn:       task.Arn,
 		ContainerName: cont.Name,
+		RuntimeID:     cont.RuntimeID,
```

Comment:
can you use `cont.GetRuntimeID()` here instead?

Reply:
yes.